packages/metrics-dotnet/MutationMetrics/Models/JsonReport.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Mutation.Metrics.Core.Models
+{
+    public struct JsonReport
```

Comment:
```suggestion
    public struct MutationTestResult
```

Since this is an implementation of the schema we should try to follow the schema as closely as possible:

https://github.com/stryker-mutator/mutation-testing-elements/blob/master/packages/report-schema/src/mutation-testing-report-schema.json